packages/angular-workspace/tsconfig.json

```diff
@@ -36,7 +36,7 @@
       "es2020",
       "dom"
     ],
-    "importsNotUsedAsValues": "error",
+    "verbatimModuleSyntax": true,
     "ignoreDeprecations": "5.0",
```

Comment:
If I remember correctly the `ignoreDeprecations` is only here because we were using `importsNotUsedAsValues` so can now be removed.